policy/HscMapper.yaml

```diff
@@ -712,7 +712,10 @@ datasets:
   # Verify metrics datasets
   colorAnalysis_verify_job:
     persistable: ignore  # Is not persistable yet
-    template: verify/color/tract-%(tract)d%(subdir)s/colorAnalysis_verify_job.json
+    template: verify/visit/tract-%(tract)d%(subdir)s/colorAnalysis_verify_job.json
   visitAnalysis_verify_job:
     persistable: ignore  # Is not persistable yet
     template: verify/visit/tract-%(tract)d%(subdir)s/visitAnalysis_verify_job.json
+  coaddAnalysis_verify_job:
+    persistable: ignore  # Is not persistable yet
+    template: verify/coadd/tract-%(tract)d%(subdir)s/coaddAnalysis_verify_job.json
```

Comment:
The directory tree needs to include the filter and, for consistency, might as well follow the format already established for the plots, but with the leading `plot/` changed to `verify/`, so this should be:
`template: verify/%(filter)s/tract-%(tract)d%(subdir)s/coaddAnalysis_verify_job.json`

While you're at it, could you also update the visit level job template above to:
`template: verify/%(filter)s/tract-%(tract)d/visit-%(visit)d%(subdir)s/visitAnalysis_verify_job.json`